src/app/components/Header/ScriptLink/index.tsx

```diff
@@ -1,19 +1,34 @@
-import React, { useContext } from 'react';
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
 import { compile } from 'path-to-regexp';
 import clone from 'ramda/src/clone';
 import { useRouteMatch } from 'react-router-dom';
-import ScriptLink from '#psammead/psammead-script-link/src';
 import { UserContext } from '#contexts/UserContext';
 import useToggle from '#hooks/useToggle';
-import { ServiceContext } from '../../../../contexts/ServiceContext';
-import { RequestContext } from '../../../../contexts/RequestContext';
+import { ServiceContext } from '../../../contexts/ServiceContext';
+import { RequestContext } from '../../../contexts/RequestContext';
+import styles from './index.styles';
+
+export type Params = Record<string, string>;
+
+interface UseRouteMatcher {
+  path: string;
+  params: Params;
+}
```

Comment:
It's tempting, I'm leaning towards not adding it. Just because this is the only instance of us using this hook, and as we slowly move over to Next.js, I don't think we'll be using this hook, we'll use the Next.js hook

Reply:
Ahh that export is actually a left over, I was importing it into the unit test, but now I'm not. I can remove the export, do you think we would still need it as a global?

Reply:
Happy to just remove the export & keep it as it, I don't want to hold you up from getting this merged.

Reply:
Done

Reply:
Actually the test was still using Params, I've just used the longhand as you suggested